packages/commons-ui-core/src/RichTypography/index.stories.js

```diff
@@ -14,7 +16,11 @@ export default {
 };
 
 function Template({ content, ...args }) {
-  return <RichTypography {...args}>{content}</RichTypography>;
+  return (
+    <Section>
+      <RichTypography {...args}>{content}</RichTypography>
+    </Section>
+  );
```

Comment:
@gertie-sheshe does it mean all stories in commons would need to be wrapped with `Section` cc @kilemensi

Reply:
No @KhadijaMahanga this is a temporary implementation to test that themes works. I will add a comment